Dalamud/Interface/Internal/Windows/Data/DataWindow.cs

```diff
@@ -235,4 +297,27 @@ private void Load()
             widget.Load();
         }
     }
+    
+    private void OnDalamudConfigurationSaved(DalamudConfiguration dalamudConfiguration)
+    {
+        this.showDebugWidgets = dalamudConfiguration.DataShowDebug;
+    }
+    
+    private bool DrawToggleButtonWithTooltip(string buttonId, string tooltip, FontAwesomeIcon icon, ref bool enabledState)
+    {
+        var result = false;
+
+        var buttonEnabled = enabledState;
```

Comment:
No need to make a new variable for this bool, it isn't modified at all (neither does enabledState need to be a ref)

Reply:
This function is copied from ConsoleWindow, it's a generic helper function, there's no real need to make it specific to this window.

A more proper solution is to move the helper function to one of the imgui helper classes, but that's out of scope for this PR.

Reply:
Additionally, copying the ref bool is important here because it is used to push/pop style variables, and the ref'd value can potentially change between the push and the pop causing a crash/corruption issue.

Reply:
But it isn't modified at all, that's why the ref and variable are both unneeded instructions

If it is modified in the original function, okay, but in this copy they aren't touched